### ISSUE TITLE
--help option sometimes skips last line of help output

### ISSUE THREAD

**Issue by siteshwar** [fish-shell/fish-shell#729]
--help option sometimes skips last line of help output. The problem can be reproduced by executing ```alias --help```.

The problem is in ```__fish_print_help``` function. It parses the manpage with nroff and then skips printing first 5 and last 3 lines of nroff output. This sometimes skips last line of output of some commands. For e.g. alias, and.

Here is the relevant code :

```
        set -l help (nroff -man "$__fish_datadir/man/man1/$item.1" ^ /dev/null )
        set -l lines (count $help)

        # Print an empty line first
        echo

        # Filter and print help
        printf "%s\n" $help| tail -n (expr $lines - 5) | head -n (expr $lines - 8) | sed $sed_cmd
```

**Comment by faho** [comment]
__fish_print_help received some major changes, is this still relevant?

**Comment by zanchey** [comment]
Yep, this was fixed with ac8c5910.
